### ISSUE TITLE
Update USDA compoent prefix in mappings

### ISSUE THREAD

**Issue by kaiiam** [CompositionalDietaryNutritionOntology/cdno#120]
The current [CDNO mapping file](https://github.com/Southern-Cross-Plant-Science/cdno/blob/master/src/ontology/templates/mappings.csv) to USDA components has misleading prefix. Currently CDNO has `USDA_NDB` however, NDB's within FoodData Central refer only to Foods not components. I'm not sure at the moment what would be the correct replacement, once USDA is able to provide one, CDNO should switch them out.

**Comment by NutrientInstitute** [comment]
I downloaded the nutrient.csv file for the April 2024 release of FoodData Central and put together the mappings. I included all previous mappings, and added some new term mappings for newer USDA terms. 

I've attached these new mappings to this comment, but these new mappings will have to be reformatted for use in the CDNO mappings template. 
[new_mappings.csv](https://github.com/CompositionalDietaryNutritionOntology/cdno/files/15153688/new_mappings.csv)
